plugins/auth-backend/api-report.md

```diff
@@ -619,6 +619,10 @@ export const providers: Readonly<{
     ) => AuthProviderFactory_2;
     resolvers: never;
   }>;
+  pinniped: Readonly<{
+    create: () => AuthProviderFactory_2;
+    resolvers: never;
+  }>;
```

Comment:
Noting that while I would've liked to not add this to `auth-backend` it's still a lil bit early for that.

Reply:
No problemo i removed the auth-provider from the default providers listed in auth-backend.